models/vrp.rb

```diff
@@ -247,19 +240,18 @@ def self.convert_position_relations(hash)
     end
 
     def self.deduce_solver_parameter(hash)
-      if hash[:configuration] && hash[:configuration][:resolution] && hash[:configuration][:resolution][:solver_parameter]
-        if hash[:configuration][:resolution][:solver_parameter] == -1
-          hash[:configuration][:resolution][:solver] = false
-        else
+      resolution = hash[:configuration] && hash[:configuration][:resolution]
+      if resolution[:solver_parameter].to_i == -1
```

Comment:
runtime error if `resolution` is `nil`

Reply:
is `to_i` necessary? if there is not `solver_parameter` `nil == -1` would correctly return false